### DISCUSSION TITLE
loss representation on graphs (vertical bar vs avg value only)

### DISCUSSION THREAD

**Discussion by meepmeep** [oetiker/SmokePing#329] [Q&A] [answered]
Hey

I'm using smokeping since few years, and it very very useful  

However, on the demo site, loss representation are shown with vertical bar taking full height of the graphs :

![demo](https://user-images.githubusercontent.com/584651/164673598-fbe7b325-8591-4c2b-8c4a-1a32704c11ed.png)

On my instance, only the average probe value get the color : 

![firefox_e9FTZPDZyV](https://user-images.githubusercontent.com/584651/164674838-d3258651-cb19-469a-921e-f8a5e9f7a10c.png)

I went through all the config file, I don't know where is this settings ..

I'm using smokeping version from the stable debian package (v2.7.3-3)

**Comment by oetiker** [comment, marked as answer]
Configure

`loss_background`

If this option is enabled, uptime data is no longer displayed in the graph background.
